website/source/docs/guides/containers-guide.md

```diff
@@ -0,0 +1,328 @@
+
+```sh
+$ docker exec -it <container_id> /bin/sh
+/ # consul members
+Node      Address          Status  Type    Build  Protocol  DC   Segment
+server-1  172.17.0.2:8301  alive   server  1.5.2  2         dc1  <all>
+client-1  172.17.0.3:8301  alive   client  1.5.2  2         dc1  <default>
+```
+
+#### Local Consul Binary
+
+If you have a local Consul binary in your PATH you can also export the `CONSUL_HTTP_ADDR` environment variable to point to the HTTP address of a remote Consul server. This will allow you to bypass `docker exec <container_id> consul <command>` and use `consul <command>` directly. 
+
+```sh
+$ export CONSUL_HTTP_ADDR=<consul_server_ip>:8500
+$ consul members
+Node      Address          Status  Type    Build  Protocol  DC   Segment
+server-1  172.17.0.2:8301  alive   server  1.5.2  2         dc1  <all>
+client-1  172.17.0.3:8301  alive   client  1.5.2  2         dc1  <default>
+```
+
+In this guide, you are binding your containerized Consul server's client address to 0.0.0.0 which allows us to communicate with our Consul cluster with a local Consul install. By default, the client address is bound to localhost.
+
+```sh
+$ which consul
+/usr/local/bin/consul
+$ consul members
+Node      Address          Status  Type    Build  Protocol  DC   Segment
+server-1  172.17.0.2:8301  alive   server  1.5.2  2         dc1  <all>
+client-1  172.17.0.3:8301  alive   client  1.5.2  2         dc1  <default>
+```
+
+### Stopping, Starting, and Restarting Containers
+
+The official Consul container supports stopping, starting, and restarting. To stop a container, run `docker stop`.
+
+```sh
+$ docker stop <container_id>
+```
+
+To start a container, run `docker start`.
+
+```sh
+$ docker start <container_id>
+```
+
+To do an in-memory reload, send a SIGHUP to the container.
+
+```sh
+$ docker kill --signal=HUP <container_id>
+```
+
+### Removing Servers from the Cluster
+
+As long as there are enough servers in the cluster to maintain [quorum](/docs/internals/consensus.html#deployment-table), Consul's [autopilot](/docs/guides/autopilot.html) feature will handle removing servers whose containers were stopped. Autopilot's default settings are already configured correctly. If you override them, make sure that the following [settings](/docs/agent/options.html#autopilot) are appropriate.
+
+* `cleanup_dead_servers` must be set to true to make sure that a stopped container is removed from the cluster.
+* `last_contact_threshold` should be reasonably small, so that dead servers are removed quickly.
+* `server_stabilization_time` should be sufficiently large (on the order of several seconds) so that unstable servers are not added to the cluster until they stabilize.
+
+If the container running the currently-elected Consul server leader is stopped, a leader election will be triggered. This event will cause a new Consul server in the cluster to assume leadership.
```

Comment:
Would suggest removing the second line, since a leader is not guaranteed to be elected. Like in the case where we don't have enough servers.

```suggestion
If the container running the currently-elected Consul server leader is stopped, a leader election will be triggered.
```